recipes/gau2grid/meta.yaml

```diff
@@ -0,0 +1,142 @@
+{% set version = "2.0.3" %}
+{% set sha256 = "bf6d6739e6548b8dbb1e190bbaaca572d7d847529f206e37def7d3553743dc4f" %}
+
+package:
+    name: {{ name|lower }}
+    version: {{ version }}
+
+source:
+    url: https://github.com/dgasmith/{{ name }}/archive/v{{ version }}.tar.gz
+    sha256: {{ sha256 }}
+
+build:
+    number: 0
+    binary_relocation: true
+    skip: true  # [py2k]
+
+requirements:
+    build:
+        - cmake >=3.0
+        - {{ compiler('c') }}
+        - ninja  # [win]
+    host:
+        - numpy
+        - python
+
+outputs:
+    - name: gau2grid
+      build:
+          run_exports:
+              - {{ pin_subpackage('gau2grid', max_pin='x') }}
+      requirements:
+          build:
+              - cmake >=3.0
+              - {{ compiler('c') }}
+              - ninja  # [win]
+          host:
+              - numpy
+              - python
+      files:
+          - include/gau2grid              # [unix]
+          - lib/libgg*                    # [unix]
+          - share/cmake/gau2grid          # [unix]
+          - Library/include/gau2grid      # [win]
+          - Library/lib/gg*               # [win]
+          - Library/share/cmake/gau2grid  # [win]
+      test:
+          commands:
+              # Verify library
+              - test -L $PREFIX/lib/libgg$SHLIB_EXT                                # [unix]
+              - if not exist %PREFIX%\\Library\\lib\\gg.lib exit 1                 # [win]
+              # Inspect linkage
+              - ldd -v $PREFIX/lib/libgg$SHLIB_EXT                                 # [linux]
+              - otool -L $PREFIX/lib/libgg$SHLIB_EXT                               # [osx]
+              - conda inspect linkages --show-files --groupby=dependency gau2grid  # [unix]
+              - conda inspect objects -p $PREFIX $PKG_NAME                         # [osx]
+      about:
+          home: https://github.com/dgasmith/gau2grid
+          dev_url: https://github.com/dgasmith/gau2grid
+          doc_url: https://github.com/dgasmith/gau2grid/blob/master/README.md
+          doc_source_url: https://github.com/dgasmith/gau2grid/blob/master/docs/source/index.rst
+          license: BSD-3-Clause
+          license_url: https://opensource.org/licenses/BSD-3-Clause
+          license_file: LICENSE
+          license_family: BSD
+          summary: "D.G.A. Smith's C library for fast computation of a Gaussian and its derivative on a grid"
+          description: >
+            A collocation code for computing gaussians on a grid of the form:
+            ```
+            out_Lp = x^l y^m z^n \sum_i coeff_i e^(exponent_i * (|center - p|)^2)
+            ```
+            , where the returned matrix dimension are the angular momentum (L) by number of requested points (p).
+
+    - name: pygau2grid
+      build:
+          ignore_run_exports:
+              # these needed as tools to build and interpreter/module to run, but generic version in both cases
+              - numpy
+              - python
```

Comment:
You can't add this since the files are in interpreter specific directory names.

Reply:
I originally added b/c it was warning that overlinked to libpython when I plain had `host: python` and `run: pin_compatible(python`. And it's true that the compiled part is pyver independent and the py part is fit to be `noarch: python`. The current state doesn't have the overlink warning and does still produce separate py36 and 37 packages, so I had thought it ok. Better to return to `host: python` and `run: pin_compatible(python`?